ui/app/components/database-role-edit.hbs

```diff
@@ -162,7 +162,7 @@
             @icon={{if this.handleCreateEditRole.isRunning "loading"}}
             type="submit"
             disabled={{this.handleCreateEditRole.isRunning}}
-            data-test-secret-save
+            data-test-save
```

Comment:
save buttons are save buttons, we don't need to make the individualized per component.